presto-spi/src/test/java/com/facebook/presto/spi/type/TestTimeZoneKey.java

```diff
@@ -191,7 +191,7 @@ public int compare(TimeZoneKey left, TimeZoneKey right)
             hasher.putShort(timeZoneKey.getKey());
             hasher.putString(timeZoneKey.getId(), StandardCharsets.UTF_8);
         }
-        // Zone file should not (normally) be changed, so let's make is more difficult
-        assertEquals(hasher.hash().asLong(), 5690324204210439335L, "zone-index.properties file contents changed!");
+        // Zone file should not (normally) be changed, so let's make this more difficult
+        assertEquals(hasher.hash().asLong(), -5839014144088293930L, "zone-index.properties file contents changed!");
```

Comment:
Does this check give any more confidence about `zone-index.properties` file immutability than review process gives anyway?